.jenkins/pytorch/test.sh

```diff
@@ -352,6 +352,21 @@ test_cpp_extensions() {
   assert_git_not_dirty
 }
 
+test_vec256() {
+  # This is to test vec256 instructions DEFAULT/AVX/AVX2 (platform dependent, some platforms might not support AVX/AVX2)
+  if [[ "$BUILD_ENVIRONMENT" != *asan* ]] && [[ "$BUILD_ENVIRONMENT" != *rocm* ]]; then
+    echo "Testing vec256 instructions"
+    pushd build/bin
+    vec256_tests=$(find . -maxdepth 1 -executable -name 'vec256_test*')
+    for vec256_exec in $vec256_tests
+    do
+      $vec256_exec
```

Comment:
Please report test results in xml format as well (so they could be parsed by CI)
```suggestion
      $vec256_exec --gtest_output=xml:test/test-reports/vec256/$vec256_exec.xml
```